FreeRTOS/Demo/Posix_GCC/main.c

```diff
@@ -152,6 +152,9 @@ static clockid_t cid = CLOCK_THREAD_CPUTIME_ID;
 
 int main( void )
 {
+    /* No buffer for stdout. */
+    setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
Formatting issue

Reply:
Formatting error corrected; And the function is added in git workflow file instead of demo main.c file